LISISO/commonfunctions.sh

```diff
@@ -315,3 +315,54 @@ function checkrpms()
 		echo "RPM's are missing"
 	fi
 }
+
+function CheckPartitionSize()
+{
+        if (( $1 < $2 ));then
+                echo "LIS installation aborted due to insufficiant space"
+                printf "\t %s \t\t %s\n" "$3" "available: $1 required: $2 Bytes"
+                return 0
+        fi
+        return 1
+}
+
+function CheckRequiredSpace()
+{
+	No_of_symvers=$(($(ls /boot | grep "symvers" | wc -l)))
```

Comment:
You may use this simple command:
No_of_symvers=$(echo /boot/symvers* | wc -w)

Also you may not use Capital first letter for variable.